http4s/src/main/scala/com.snowplowanalytics.snowplow.collectors.scalastream/CollectorService.scala

```diff
@@ -109,7 +107,8 @@ class CollectorService[F[_]: Sync](
       ).flatten
       responseHeaders = Headers(headerList)
       _ <- sinkEvent(event, partitionKey)
-    } yield buildHttpResponse(responseHeaders, pixelExpected)
+      resp = buildHttpResponse(responseHeaders, pixelExpected)
+    } yield resp
```

Comment:
Any specific reason why not inline like before? :)

Reply:
No specific reason, I just like this more :D